sdcm/nemesis.py

```diff
@@ -100,6 +101,13 @@
 from test_lib.cql_types import CQLTypeBuilder
 
 LOGGER = logging.getLogger(__name__)
+# NOTE: following lock is needed in the K8S multitenant case
+NEMESIS_LOCK = Lock()
+NEMESIS_RUN_INFO = {}
+EXCLUSIVE_NEMESIS_NAMES = (
```

Comment:
didn't we discussed those to be labeled, and then we could enable/disable them, and make sure we only run one of them by the labels ? i.e. not introducing a new mechanize to make them exclusive

Reply:
The current approach, I think, is pretty stable and doesn't require us to configure it anyhow.
Then the list of exclusive nemesis is going to be stable. Only K8S node disruptions of this type.